manifest/docker_schema2_list.go

```diff
@@ -81,7 +81,7 @@ func (list *Schema2List) UpdateInstances(updates []ListUpdate) error {
 		if updates[i].MediaType == "" {
 			return errors.Errorf("update %d of %d passed to Schema2List.UpdateInstances had no media type (was %q)", i+1, len(updates), list.Manifests[i].MediaType)
 		}
-		if err := SupportedSchema2MediaType(updates[i].MediaType); err != nil && SupportedOCI1MediaType(updates[i].MediaType) != nil {
```

Comment:
Removing the OCI checks while keeping the schema2 ones is causing the CI failures:

time="2020-01-27T23:43:21Z" level=fatal msg="Error updating manifest list: update 1 of 6 passed to Schema2List.UpdateInstances had an unsupported media type (was \"application/vnd.docker.distribution.manifest.v2+json\"): \"application/vnd.oci.image.manifest.v1+json\": unsupported docker v2s2 media type: \"application/vnd.oci.image.manifest.v1+json\""

I guess we have to relax the checks altogether.

Reply:
If we are relaxing the schema2 checks _here_, why not do it in the other places as well (i.e. do the same edits for `SupportedSchema2MediaType`)? That way OCI and schema2 remain consistent.